python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -356,142 +360,144 @@ tables:
     datatype: float
   - name: z_hsm_moments_31
-    description: HSM higher order moments 31. Measured on z-band.
+    description: HSM higher-order moment 31. Measured on z-band.
     datatype: float
   - name: z_hsm_momentsPsf_31
-    description: HSM higher order PSF moments 31. Measured on z-band.
+    description: HSM higher-order moment 31 measured on the PSF model at the position of the object. Measured on z-band.
     datatype: float
   - name: z_hsm_moments_22
-    description: HSM higher order moments 22. Measured on z-band.
+    description: HSM higher-order moment 22. Measured on z-band.
     datatype: float
   - name: z_hsm_momentsPsf_22
-    description: HSM higher order PSF moments 22. Measured on z-band.
+    description: HSM higher-order moment 22 measured on the PSF model at the position of the object. Measured on z-band.
     datatype: float
   - name: z_hsm_moments_13
-    description: HSM higher order moments 13. Measured on z-band.
+    description: HSM higher-order moment 13. Measured on z-band.
     datatype: float
   - name: z_hsm_momentsPsf_13
-    description: HSM higher order PSF moments 13. Measured on z-band.
+    description: HSM higher-order moment 13 measured on the PSF model at the position of the object. Measured on z-band.
     datatype: float
   - name: z_hsm_moments_04
-    description: HSM higher order moments 04. Measured on z-band.
+    description: HSM higher-order moment 04. Measured on z-band.
     datatype: float
   - name: z_hsm_momentsPsf_04
-    description: HSM higher order PSF moments 04. Measured on z-band.
+    description: HSM higher-order moment 04 measured on the PSF model at the position of the object. Measured on z-band.
     datatype: float
   - name: z_hsm_moments_flag
-    description: General failure flag, to be used in conjunction with z_i_flag. Measured
+    description: Flag set for any failure in the HSM higher-order moments. Measured
       on z-band.
     datatype: boolean
   - name: z_hsm_momentsPsf_flag
-    description: General failure flag, to be used in conjunction with z_iPSF_flag.
+    description: Flag set for any failure in the HSM higher-order moment on the PSF model.
       Measured on z-band.
     datatype: boolean
   - name: z_inputCount
-    description: Number of images contributing at center, not including anyclipping.
+    description: Number of images contributing at center, not including any clipping.
       Forced on z-band.
     datatype: int
   - name: z_deblend_dataCoverage
     description: Fraction of data that contained good data, ie. 1 - number of no data
       pixels/total number of pixels in the z-band.
     datatype: float
   - name: z_deblend_blendedness
-    description: Blendedness in the deconvolved scarlet space
+    description: Blendedness in the deconvolved scarlet space.
     datatype: float
   - name: z_deblend_fluxOverlap
```

Comment:
Should this have units of flux (`nJy`)?

Reply:
No, blendedness is unitless (it's a ratio of the source flux to the flux of the entire blend model).